guides/common/assembly_restoring-server-or-smart-proxy-from-a-backup.adoc

```diff
@@ -2,8 +2,8 @@ include::modules/con_restoring-server-or-smart-proxy-from-a-backup.adoc[]
 
 include::modules/proc_restoring-from-a-full-backup.adoc[leveloffset=+1]
 
-include::modules/proc_restoring-from-incremental-backups.adoc[leveloffset=1]
+include::modules/proc_restoring-from-incremental-backups.adoc[leveloffset=+1]
 
-include::modules/con_backup-and-restore-smart-proxy-using-a-virtual-machine-snapshot.adoc[leveloffset=1]
+include::modules/con_backup-and-restore-smart-proxy-using-a-virtual-machine-snapshot.adoc[leveloffset=+1]
 
-include::modules/proc_synchronizing-an-external-smart-proxy.adoc[leveloffset=2]
+include::modules/proc_synchronizing-an-external-smart-proxy.adoc[leveloffset=+2]
```

Comment:
This one looks out of place? Anything three levels down will be buried. 
10.3.1. Synchronizing an External Smart Proxy

Reply:
Yes, but IHMO it looks off on nightly:

```
[10. Restoring Foreman server or Smart Proxy server from a Backup](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_Server_or_Smart_Proxy_from_a_Backup_admin)
[10.1. Restoring from a Full Backup](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_from_a_Full_Backup_admin)
[11. Restoring from Incremental Backups](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_from_Incremental_Backups_admin)
[12. Backup and Restore Smart Proxy server Using a Virtual Machine Snapshot](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Backup_and_Restore_Smart_Proxy_Using_a_Virtual_Machine_Snapshot_admin)
[12.1. Synchronizing an External Smart Proxy](https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Synchronizing_an_External_Smart_Proxy_admin)
```

vs on my feature branch:

```
[10. Restoring Foreman server or Smart Proxy server from a Backup](https://github.com/theforeman/foreman-documentation/pull/1077#Restoring_Server_or_Smart_Proxy_from_a_Backup_admin)
[10.1. Restoring from a Full Backup](https://github.com/theforeman/foreman-documentation/pull/1077#Restoring_from_a_Full_Backup_admin)
[10.2. Restoring from Incremental Backups](https://github.com/theforeman/foreman-documentation/pull/1077#Restoring_from_Incremental_Backups_admin)
[10.3. Backup and Restore Smart Proxy server Using a Virtual Machine Snapshot](https://github.com/theforeman/foreman-documentation/pull/1077#Backup_and_Restore_Smart_Proxy_Using_a_Virtual_Machine_Snapshot_admin)
```

There are many instances where we have content that is not accessible from the navigation which is absolutely fine in my eyes. Our guides and their navs are already long enough.

I view this change as an improvement.